src/client/src/core/pages/ModuleUnderDevelopment.vue

```diff
@@ -31,15 +31,15 @@
           alt="this module is under development"
         />
         <oxd-text tag="p" class="orangehrm-module-under-development-text">
-          {{ $t('general.this_page_is_being_developed', {version: '5.4'}) }}
+          {{ $t('general.page_under_development') }}
```

Comment:
seems `this_page_is_being_developed` and `download_latest_release_with_all_features` unused strings not deleted

e.g
```php
        $this->getLangHelper()->deleteLangStringByUnitId(
            'this_page_is_being_developed',
            $this->getLangHelper()->getGroupIdByName('general')
        );
```